### PR TITLE
Created Profile and Settings Page

### PR DESCRIPTION
**Pull request by tyang98**
### Summary 
This pull request creates the profile and settings page for the Rider app, as well as adds Google Authentication to retrieve information from user.

### Notes 

Need to fix the logout button, because it currently does not remove the user login from the app.

### Screenshots 
Login
Profile
Settings

### CODE REVIEW

File: Rider/app/src/main/java/com/dti/rider/ProfileActivity.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import de.hdodenhof.circleimageview.CircleImageView
+import kotlinx.android.synthetic.main.activity_profile.*
+
+
+class ProfileActivity : AppCompatActivity() {
+
+    lateinit var mGoogleSignInClient : GoogleSignInClient
+    lateinit var name : TextView
+    lateinit var email : TextView
```
Comment (SamChou19815) [inline Rider/app/src/main/java/com/dti/rider/ProfileActivity.kt]: Do these fields need to be public?

**Review by SamChou19815 — CHANGES_REQUESTED** [review]
Problems

File: Rider/app/src/main/java/com/dti/rider/ProfileActivity.kt (hunk 1 above)
Comment (tyang98) [reply Rider/app/src/main/java/com/dti/rider/ProfileActivity.kt]: No they do not have to be.

**Comment by SamChou19815** [comment]
@tyang98 Could you deal with merge conflicts first?

**Pull request closed without merging**